metadata_annotation.ipynb

```diff
@@ -0,0 +1,2466 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    # subset the data
</pre><p>Why? Mention.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>